src/EventTeachingService.cpp

```diff
@@ -407,6 +415,12 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
           controller->sendCMDERR(CMDERR_INVALID_EVENT);
           return PROCESSED;
         }
+        
+        if (evindex > module_config->EE_NUM_EVS)
+        { controller->sendCMDERR(CMDERR_INV_EV_IDX);
```

Comment:
New line please after the curly bracket.

Reply:
Done